service TestService {
    i32 returnInt(1: i32 intParameter, 2: string stringParameter)
    void voidMethod()
}